<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-1HND23K" tests="6" failures="2" timestamp="8 Sep 2017 21:10:26 GMT" time="0.015" errors="0">
  <testcase name="testSumDouble" time="0.0" classname="SumTest">
    <failure type="java.lang.AssertionError" message="1.2+8.6 should be 9.8 expected [9.8] but found [9.799999999999999]">
      <![CDATA[java.lang.AssertionError: 1.2+8.6 should be 9.8 expected [9.8] but found [9.799999999999999]
	at SumTest.testSumDouble(SumTest.java:29)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testSumDouble -->
  <testcase name="testSumLong" time="0.0" classname="SumTest"/>
  <testcase name="testSumLong" time="0.0" classname="SumTest"/>
  <testcase name="testSumLong" time="0.0" classname="SumTest"/>
  <testcase name="testSubDouble" time="0.0" classname="SubTest">
    <failure type="java.lang.AssertionError" message="1.2-8.6 should be -7.4 expected [-7.4] but found [-7.3999999999999995]">
      <![CDATA[java.lang.AssertionError: 1.2-8.6 should be -7.4 expected [-7.4] but found [-7.3999999999999995]
	at SubTest.testSubDouble(SubTest.java:19)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testSubDouble -->
  <testcase name="testSubLong" time="0.0" classname="SubTest"/>
</testsuite>
